buoyant_cloud/manifest.json

```
@@ -0,0 +1,95 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dee4b74f-34b7-457e-98b1-7bb8306f2c18",
+  "app_id": "buoyant-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "uninstallation": "README.md#Uninstallation",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Buoyant Cloud provides fully managed Linkerd, right on your cluster.",
+    "title": "Buoyant Cloud",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Buoyant Cloud: Datadog dashboard",
+        "image_url": "images/bcloud_datadog_dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Buoyant Cloud: Overview page",
+        "image_url": "images/bcloud_01.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Buoyant Cloud: Linkerd health view",
+        "image_url": "images/bcloud_02.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Buoyant Cloud: Managed Linkerd event",
+        "image_url": "images/bcloud_03.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Buoyant Cloud: Traffic page",
+        "image_url": "images/bcloud_04.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Buoyant Cloud: Topology page",
+        "image_url": "images/bcloud_05.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Buoyant Cloud: Metrics page",
+        "image_url": "images/bcloud_06.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Buoyant Cloud: Workload Detail page",
+        "image_url": "images/bcloud_07.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Cloud",
+      "Category::Marketplace",
+      "Category::Monitoring",
+      "Category::Network",
+      "Category::Security",
+      "Offering::Integration",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "Buoyant Cloud",
+      "configuration": {},
+      "events": {
+        "creates_events": true
+      },
+      "metrics": {
+        "prefix": "buoyant_cloud.",
+        "check": [],
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "dashboards": {
+      "Buoyant Cloud": "assets/dashboards/buoyant_cloud_overview.json"
+    }
+  },
+  "author": {
+    "support_email": "cloud@buoyant.io",
+    "name": "Andrew Seigner",
```

```suggestion
    "name": "Buoyant",
```
Sorry, should be the last thing - but this should by the company's name, not yours.